src/main/java/games/strategy/util/PropertyUtil.java

```diff
@@ -116,10 +116,10 @@ private static Method getSetter(final String propertyName, final Object subject,
         try {
           final Class<?> argType = value.getClass();
           return subject.getClass().getMethod(setterName, argType);
-        } catch (final NoSuchMethodException nsmf) {
-          // Go ahead and try the first one
-          return m;
-        } catch (final NullPointerException n) {
+        } catch (final NoSuchMethodException | NullPointerException nsmf) {
```

Comment:
Might be appropriate to rename `nsmf` to just `e` here since it no longer represents just a `NoSuchMethodException`.

(Same comment for the nearly identical block on line 137 below.)